pkg/reconciler/knativeeventing/knativeeventing.go

```diff
@@ -123,6 +125,7 @@ func (r *Reconciler) reconcile(ctx context.Context, ke *eventingv1alpha1.Knative
 		r.ensureFinalizer,
 		r.initStatus,
 		r.install,
+		r.checkJobs,
```

Comment:
I actually would like to see a new transformer to convert the Job named `storage-version-migration`.
@jcrossley3 what do u think?